SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+The GeoNetwork community takes the security of the software and all services based on the software product seriously. On this page you can find the versions for which the community provides security patches. 
+
+If you believe you have found a security vulnerability in the software or an implementation of the software, please report it to geonetwork@osgeo.org as described below. Do not publish the vulnerability in any public forums.
+
+## Supported Versions
+
+Each GeoNetwork release is supported with bug fixes for a limited period, with patch releases made approximately every three to six  months. We recommend to update to latest patch release as soon as possible to patch any vulnarabilities.
```

Comment:
Recommend major / minor / incremental version numbers.

> Each GeoNetwork minor release is supported with bug fixes for a limited period, with incremental “patch” releases made approximately every three to six  months.
* We recommend to update to latest incremental release as soon as possible to address security vulnarabilities.
* Some overlap is provided when major versions are announced with both a current version and a maintenance version being made available to provide time for organizations to upgrade.